contracts/Staking/Proxy/ProxyEscrowThales.sol

```diff
@@ -0,0 +1,190 @@
+pragma solidity ^0.5.16;
+
+import "openzeppelin-solidity-2.3.0/contracts/math/Math.sol";
+import "synthetix-2.50.4-ovm/contracts/SafeDecimalMath.sol";
+import "openzeppelin-solidity-2.3.0/contracts/token/ERC20/SafeERC20.sol";
+
+import "../../utils/proxy/ProxyReentrancyGuard.sol";
+import "../../utils/proxy/ProxyOwned.sol";
+import "../../utils/proxy/ProxyPausable.sol";
+import "@openzeppelin/upgrades-core/contracts/Initializable.sol";
+
+import "../../interfaces/IEscrowThales.sol";
+import "../../interfaces/IStakingThales.sol";
+
+contract ProxyEscrowThales is IEscrowThales, Initializable, ProxyOwned, ProxyReentrancyGuard, ProxyPausable {
```

Comment:
Please delete EscrowThales.sol if its not going be to be used anymore.

Reply:
the interface needs to stay. 
But yes, the EscrowThales can be deleted.